components/03-forms/form.njk

```diff
@@ -0,0 +1,20 @@
+<form{% if formModifier %} class="{{ formModifier }}"{% endif %}>
+  <fieldset class="dso-form-fields">
+    <legend class="sr-only">{{ formLegend }}</legend>
+    {% for outerfieldset in fieldsets %}
```

Comment:
Is er een reden dat je niet voor `fieldset in fieldsets` hebt gekozen? Dat je `outerfieldset` (wat eigenlijk outerFieldset moet zijn - 2 woorden) is er geen variable `innerFieldset`

Reply:
aangepast